wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallController/ActiveCallRouter.swift

```diff
@@ -100,9 +100,9 @@ extension ActiveCallRouter: ActiveCallRouterProtocol {
         // NOTE: We resign first reponder for the input bar since it will attempt to restore
         // first responder when the call overlay is interactively dismissed but canceled.
         UIResponder.currentFirst?.resignFirstResponder()
-
+        guard let zclient = zClientViewController else { return }
```

Comment:
Can we inject the user session instead of getting the zclientviewcontroller?

Reply:
Sure! Just fixed that!